.travis.yml

```diff
@@ -2,15 +2,38 @@ language: dart
 
 sudo: false
 
-dart:
-  - stable
-  - "1.24.3"
-  - dev
+branches:
+  only:
+    - master
+
+jobs:
+  include:
+    - stage: Repo
+      name: 'configlet lint'
+      script:
+        - ./bin/fetch-configlet
+        - ./bin/configlet version
+        - ./bin/configlet lint --track-id dart .
+      language: bash
+
+    - stage: Exercises
+      name: 'anagram'
+      script: ./bin/travis.sh
+      env: EXERCISE_DIR="anagram"
+      dart: stable
+      install: true
+
+    - stage: Exercises
+      name: 'armstrong_numbers'
+      script: ./bin/travis.sh
+      env: EXERCISE_DIR="armstrong-numbers"
+      dart: stable
+      install: true
+
+stages:
+  - Repo
+  - Exercises
```

Comment:
Neither stage Repo nor Exercises set 1.24.3 or dev to be run.

Those versions of Dart should be included so we can see how handling multiple versions of Dart would work.